src/product-service/data.json

```diff
@@ -0,0 +1,32 @@
+[
```

Comment:
I would recomment the following folders structure

src/data/products.json

Reply:
Done